core/templates/pages/exploration-editor-page/editor-tab/templates/modal-templates/customize-interaction-modal.component.ts

```diff
@@ -254,10 +256,32 @@ export class CustomizeInteractionModalComponent
   }
 
   isSaveInteractionButtonEnabled(): boolean {
-    return !!(
-      this.hasCustomizationArgs &&
-      this.stateInteractionIdService.displayed &&
-      (this.getCustomizationArgsWarningsList().length === 0));
+    let explorationTitle = this.getTitle(
+      this.stateInteractionIdService.displayed);
+    let EndExploration = 'End Exploration';
+    /*
+      There is a condition for the END EXPLORATION interaction
+      We have to check that the input field for adding
+      the exploration Ids must be non-empty
+      Save Interaction button should be disable for empty input
+    */
+    if (explorationTitle === EndExploration) {
+      let inputField = document.getElementsByClassName('exploration-id-input');
+      if (inputField === undefined) {
+        return true;
+      }
+      let inputValue = angular.element(
+        inputField[inputField.length - 1]).attr('ng-reflect-model');
+      if (inputValue !== '') {
+        return true;
+      }
+    } else {
+      return !!(
```

Comment:
Why this double negation?

Reply:
this else portion was not done by me.. It was already written by someone else, I just added that if condition for End Exploration! Although, double negotiation turns a "truthy" or "falsy" value into a Boolean value, true or false